### PR TITLE
Extra optional fields

### PR DESCRIPTION
**Pull request by karan-payfirma**

### CODE REVIEW

File: demo-reactjs/src/App.js [hunk 1]
```diff
@@ -33,7 +33,16 @@ class App extends Component {
       // State that needs to be contained with redirect URL when the transaction is done
       state: 'stateIWantToKeep',
       // Field used to keep track of transactions and for future accounting reconciliation
-      custom_id: 'Please enter your custom Id'
+      custom_id: 'Please enter your custom Id',
+            
+      // Extra optional fields can be sent below
+
+      // firstName: 'firstName',
```
Comment (anmolpayfirma) [inline demo-reactjs/src/App.js:40]: key for this field is incorrect, should be 'first_name' instead

File: demo-reactjs/src/App.js [hunk 2]
```diff
@@ -33,7 +33,16 @@ class App extends Component {
...
+      // lastName: 'lastName',
```
Comment (anmolpayfirma) [inline demo-reactjs/src/App.js:41]: key for this field is incorrect, should be 'last_name' instead

**Review by anmolpayfirma — CHANGES_REQUESTED** [review]
Please review the comments

**Comment by karan-payfirma** [comment]
> Please review the comments

Fixed it.

**Review by anmolpayfirma — APPROVED** [review]

**Pull request merged**